From cce1563af8889e14469642e926b6990a9132c2e0 Mon Sep 17 00:00:00 2001
From: Bhaskar Upadhaya <Bhaskar.Upadhaya@nxp.com>
Date: Tue, 9 Jan 2018 13:03:40 +0530
Subject: [PATCH 107/129] board: ls1012a2g5rdb: avoid phy reset

ls1012a2g5rdb does not have requirement of phy reset, so placing
reset_phy code for ls1012ardb only.

Signed-off-by: Bhaskar Upadhaya <Bhaskar.Upadhaya@nxp.com>
---
 board/freescale/ls1012ardb/eth.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/board/freescale/ls1012ardb/eth.c b/board/freescale/ls1012ardb/eth.c
index 713f830..b1fc5fa 100644
--- a/board/freescale/ls1012ardb/eth.c
+++ b/board/freescale/ls1012ardb/eth.c
@@ -23,6 +23,7 @@
 
 void reset_phy(void)
 {
+#ifdef CONFIG_TARGET_LS1012ARDB
 	/* Through reset IO expander reset both RGMII and SGMII PHYs */
 	i2c_reg_write(I2C_MUX_IO2_ADDR, 6, __PHY_MASK);
 	i2c_reg_write(I2C_MUX_IO2_ADDR, 2, __PHY_ETH2_MASK);
@@ -31,6 +32,7 @@ void reset_phy(void)
 	mdelay(10);
 	i2c_reg_write(I2C_MUX_IO2_ADDR, 2, 0xFF);
 	mdelay(50);
+#endif
 }
 
 int board_eth_init(bd_t *bis)
-- 
1.7.1

